src/Player.ts

```diff
@@ -993,22 +993,19 @@ export class Player {
     }
   }
 
+  public isHowToPay(u: unknown): u is HowToPay {
+    if (typeof u !== 'object') return false;
+    if (!u) return false;
+    const h = u as {[key in keyof HowToPay]?: any};
```

Comment:
```suggestion
    const h = u as {[key in keyof HowToPay]: number};
```

This would avoid a usage of `any`.